vault/src/execution.rs

```diff
@@ -164,7 +175,16 @@ impl Request {
             .wait_for_transaction_metadata(txid, BITCOIN_MAX_RETRYING_TIME, num_confirmations)
             .await?;
 
-        tracing::info!("Bitcoin successfully sent to {}", recipient);
+        tracing::info!("Awaiting parachain confirmations...");
+
+        provider
+            .wait_for_block_in_relay(
+                H256Le::from_bytes_le(&tx_metadata.block_hash.to_vec()),
+                Some(num_confirmations),
+            )
+            .await?;
+
+        tracing::info!("Bitcoin successfully sent and relayed");
```

Comment:
minor nit: the recipient is quite interesting in the logs, but not blocking for this PR